clean.go

```diff
@@ -196,14 +196,20 @@ func cleanUntracked() error {
 		}
 
 		dir := filepath.Join(config.BuildDir, file.Name())
-		if err := show(passToGit(dir, "clean", "-fx")); err != nil {
-			return err
+		if isGitRepository(dir) {
+			if err := show(passToGit(dir, "clean", "-fx")); err != nil {
+				return err
+			}
 		}
 	}
-
 	return nil
 }
 
+func isGitRepository(dir string) bool {
+	cmd := passToGit(dir, "rev-parse")
```

Comment:
Couldn't we just do a 
```go
_, err := os.Stat(filepath.Join(dir, ".git"))
return  !os.IsNotExist(err)
```

To check the existence of a `.git`? It's not as perfect as letting git check but it would avoid shelling out for it